ansible/configs/osp-sandbox/destroy_env.yml

```diff
@@ -0,0 +1,34 @@
+---
+- import_playbook: ../../setup_runtime.yml
+
+- name: Teardown OpenStack project and resources
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  become: false
+  tasks:
+    - name: Run infra-osp-dns
+      include_role:
+        name: infra-osp-dns
+      vars:
+        _dns_state: absent
+
+    - name: Remove DNS entry for OpenShift API and ingress
+      nsupdate:
+        server: "{{ osp_cluster_dns_server }}"
+        zone: "{{ osp_cluster_dns_zone }}"
+        record: "{{ item }}.{{ guid }}"
+        type: A
+        key_name: "{{ ddns_key_name }}"
+        key_secret: "{{ ddns_key_secret }}"
+        state: absent
```

Comment:
You need to add the port here too, no ?

Reply:
Good catch...we dont' need this at all since we don't create any OCP specific DNS entries. The common DNS role should be all we need.

Update pushed.